src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -14,55 +15,94 @@ internal static class PasswordGenerator
 
     internal const string LowerCaseChars = "abcdefghjkmnpqrstuvwxyz"; // exclude i,l,o
     internal const string UpperCaseChars = "ABCDEFGHJKMNPQRSTUVWXYZ"; // exclude I,L,O
-    internal const string LettersChars = LowerCaseChars + UpperCaseChars;
-    internal const string DigitChars = "0123456789";
+    internal const string NumericChars = "0123456789";
     internal const string SpecialChars = "-_.{}~()*+!"; // exclude &<>=;,`'^%$#@/:[]
 
     /// <summary>
-    /// Creates a cryptographically random password.
+    /// Creates a cryptographically random string.
     /// </summary>
-    /// <param name="lowerCase">The number of lowercase chars in the generated password or 0 to ignore this component.</param>
-    /// <param name="upperCase">The number of uppercase chars in the generated password or 0 to ignore this component.</param>
-    /// <param name="digit">The number of digits in the generated password or 0 to ignore this component.</param>
-    /// <param name="special">The number of special chars in the generated password or 0 to ignore this component.</param>
-    /// <returns>A cryptographically random password.</returns>
-    /// <exception cref="ArgumentOutOfRangeException">If lowerCase, upperCase, digit or special is negative or their sum is zero.</exception>
-    public static string GeneratePassword(int lowerCase, int upperCase, int digit, int special)
+    public static string Generate(int minLength,
```

Comment:
@danmoseley I believe this addresses your feedback from https://github.com/dotnet/aspire/pull/2627#discussion_r1511951003. PTAL.

Reply:
Yes